src/cloudai/_core/test_scenario_parser.py

```diff
@@ -136,8 +139,14 @@ def _parse_data(self, data: Dict[str, Any]) -> TestScenario:
         total_weight = sum(tr.weight for tr in ts_model.tests)
         normalized_weight = 0 if total_weight == 0 else 100 / total_weight
 
+        prologue, epilogue = None, None
+        if ts_model.prologue:
+            prologue = self.plugin_mapping.get(ts_model.prologue)
```

Comment:
Should we handle situation when specified name doesn't exist in the mapping?

Reply:
I chose not to raise an exception when plugins are missing. The plugins will not be executed, and I have added warning messages.

Reply:
This is about meeting expectations, ours and our users'. Warning will likely go unnoticed, so we should answer to ourselves:
1. If we raise an error here, it will make the plugin feature more visible to users, while we prefer it to "just work".
2. However, plugins are meant to save time on costly runs. If we let it proceed without plugins when they’re enabled, it won’t fulfil its intended purpose.

Reply:
I believe warning messages are sufficient. We hard-coded the plugin path, with the intent to support plugins silently whenever possible. If the plugin directories do not exist, plugins should be skipped without raising an exception. The system should function as expected even if plugins are missing.

Reply:
cc: @srinivas212 , @srivatsankrishnan